### PR TITLE
ENG-7021 - Capture Gyro/Accel on cadence

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

File: NeuroID/NeuroIDClass/Extensions/NIDSend.swift [hunk 1]
```diff
@@ -50,6 +50,39 @@ internal extension NeuroID {
         return workItem
     }
 
+    static func initGyroAccelCollectionTimer() {
+        if let workItem = NeuroID.sendGyroAccelCollectionWorkItem {
+            // Send up the first payload, and then setup a repeating timer
+            DispatchQueue
+                .global(qos: .utility)
+                .asyncAfter(
+                    deadline: .now() + GYRO_SAMPLE_INTERVAL, // 200 ms
```
Comment (priya-xavier) [inline NeuroID/NeuroIDClass/Extensions/NIDSend.swift:59]: Does the .now() return time in seconds or nanoseconds? Wasn't able to tell from the docs.

**Review by priya-xavier — DISMISSED** [review]

**Comment by kishisaka-nid** [comment]
Pulled branch into storyboard sample app and run , it seems to be printing this debug in logs constantly (~10 times a second). Is this expected?
```
(NeuroID Info)  ["NeuroID post to API Successful"]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
(NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
(NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
```

**Comment by kevinsites** [comment]
> Pulled branch into storyboard sample app and run , it seems to be printing this debug in logs constantly (~5 times a second). Is this expected?
> 
> ```
> (NeuroID Info)  ["NeuroID post to API Successful"]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> (NeuroID Debug) NeuroID IH: ["Adding NeuroID Health Event CADENCE_READING_ACCEL"]
> (NeuroID Debug) Event: ["CADENCE_READING_ACCEL - NO_TARGET - accel= gyro="]
> ```

Yes this is expected for the time being. Once we finalize with DS on what the cadence should be then we can handle logs better

**Review by kishisaka-nid — APPROVED** [review]
Pulled, tested, LGTM. 🚢

**Review by priya-xavier — APPROVED** [review]

File: NeuroID/NeuroIDClass/Extensions/NIDSend.swift (hunk 1 above)
Comment (priya-xavier) [reply NeuroID/NeuroIDClass/Extensions/NIDSend.swift:59]: Looks like .now() is in nanoseconds but adding seconds to it in here seems to be working as expected.

**Pull request merged**